### ISSUE TITLE
set up string_decoder

### ISSUE THREAD

**Issue by mcollina** [nodejs/github-bot#166]
How can I get the webhook configured? I do not have access to the secret and all the folks that I pinged in @nodejs/build do not know about that specific step. 

Here is the repo: https://github.com/nodejs/string_decoder.

**Comment by phillipj** [comment]
There's two things needed in the following order;

1. temporarily activate Travis CI to get the connection between github.com and Travis created
2. custom webhook configured telling the @nodejs-github-bot to start polling Travis CI statuses continuously when new commits are pushed to any branch

Step 1 is usually the part that has been troublesome and caused lots of discussions over the years, as it requires us to allow Travis CI private access to the nodejs organisation for a short while. Those permissions may be revoked after the initial Travis CI webhook has been created.

The Travis CI connection has to be in place before thinking about setting up the custom webhook towards @nodejs-github-bot.

Since string_decoder is not listed in [Travis CI's list of "connected repos"](https://travis-ci.org/nodejs), I assume step 1 has not been done for that repo yet. That's done by adding the "Travis CI" service in the repo settings -> Integrations & services:

<img width="1010" alt="screen shot 2018-03-01 at 22 30 48" src="https://user-images.githubusercontent.com/1231635/36870703-38e68e5a-1da0-11e8-9910-4c084d11bc2e.png">

It's been a while since I tried to activate Travis CI in a nodejs repo, but historically that flow ends in one having to request permissions from the nodejs org owners, to allow org wide private access. Whether or not the org owners are positive to that these days, I don't know to be honest, might be less of an issue these days now that the private nodejs org has been created.

As soon as we know that Travis CI connection is up-n-running, adding the custom bot webhook is a walk in the park. I'd suggest setting up the webhook as configured in the [nodejs.org repo](https://github.com/nodejs/nodejs.org) and ping me, @joyeecheung or @rvagg to get the webhook secret needed.

Does that make sense?

**Comment by mcollina** [comment]
Step 1 was already done when I opened this.

<img width="1052" alt="schermata 2018-03-01 alle 22 48 49" src="https://user-images.githubusercontent.com/52195/36871522-ca0c6682-1da2-11e8-90fa-acce3411905d.png">

I just need the secret (and it's probably better if someone else set it up).

**Comment by mcollina** [comment]
I have been following this guide https://github.com/nodejs/admin/pull/68. And I'm stuck in adding the secret.

**Comment by phillipj** [comment]
Alright, I'll ping you in IRC with the secret, as I don't have enough permissions in the string_decoder repo to create it myself.

**Comment by mcollina** [comment]
@phillipj awesome! thank you! I'll send the PR here tomorrow to add the repo to the bot.
